### PR TITLE
feat: add get all articles endpoint

### PR DESCRIPTION
**Pull request by IscoRuta**
# Pull Request

## Scope
https://dev.azure.com/CPerrie/Aarons%20Kit%20-%20Information%20Storage/_workitems/edit/47
## Work Done
Changed journals.py to articles.py.
Changed the app route  to "/articles"
Updated the SQL statements so that it outputs all the articles in the MySQL database.
## Steps to Test
Run the app through docker (docker compose up)
type 'localhost:5000/articles' in your web browser
The metadata of the dummy articles should appear (see example below).
metadata

### CODE REVIEW

File: app/rest.py [hunk 1]
```diff
@@ -2,3 +2,5 @@
 
 if __name__ == "__main__":
     app.run(debug=True, host="0.0.0.0")
+
+import journals
```
Comment (DominicGBauer) [inline app/rest.py]: what is journals, should this not be articles?

**Review by DominicGBauer — CHANGES_REQUESTED** [review]

File: app/rest.py (hunk 1 above)
Comment (tumelomakgaka) [reply app/rest.py]: Makes sense Dom, will make the change

**Review by tumelomakgaka — APPROVED** [review]
Looks good to me

**Review by DominicGBauer — APPROVED** [review]

File: app/rest.py (hunk 1 above)
Comment (CailinPerrie) [reply app/rest.py]: These changes were also made to the database schema

**Pull request merged**